packages/ui-library/src/components/form-label/index.stories.ts

```diff
@@ -92,9 +92,9 @@ export default {
         The Form Label is intended to be used when creating new components. Currently, it is used like this in the following components:
         
         - [**Checkbox**](?path=/docs/components-checkbox--docs)
-        - [**Number Input**](?path=/docs/components-number-input--docs)
+        - [**Input Field Number**](?path=/docs/components-input-field-number--docs)
         - [**Select**](?path=/docs/components-select--docs)
-        - [**Text Input**](?path=/docs/components-text-input--docs)
+        - [**Input Field Text**](?path=/docs/components-input-field-text--docs)
```

Comment:
Could you move this line up so that the component names are ordered alphabetically?